app/pulp/app/models/task.py

```diff
@@ -145,7 +145,7 @@ class Task(Model):
     started_at = models.DateTimeField(null=True)
     finished_at = models.DateTimeField(null=True)
 
-    non_fatal_errors = JSONField()
+    non_fatal_errors = JSONField(default=[])
```

Comment:
I think this will result in a classic python bug. Exactly one list instance will be created and used as the default forever, and it'll be handed out to anyone who needs it. If something adds stuff to it, that stuff will still be there the next time the default gets used.

You can specify a callable as the default, which will be called each time a default value is required. So you can probably say ``default=list``.